test/network/NetworkSettings.ts

```diff
@@ -179,13 +181,21 @@ describe('NetworkSettings', () => {
                 const reserveToken2 = await createTestToken();
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.false;
                 expect(await networkSettings.isTokenWhitelisted(reserveToken2.address)).to.be.false;
+                expect(await networkSettings.protectedTokenWhitelist()).not.to.have.members([
+                    reserveToken.address,
+                    reserveToken2.address
+                ]);
 
                 const res = await networkSettings.addTokensToWhitelist([reserveToken.address, reserveToken2.address]);
                 await expect(res).to.emit(networkSettings, 'TokenAddedToWhitelist').withArgs(reserveToken.address);
                 await expect(res).to.emit(networkSettings, 'TokenAddedToWhitelist').withArgs(reserveToken2.address);
 
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.true;
                 expect(await networkSettings.isTokenWhitelisted(reserveToken2.address)).to.be.true;
+                expect(await networkSettings.protectedTokenWhitelist()).to.have.members([
```

Comment:
`to.deep.equal` would be a little more restrictive.

Reply:
The test only checks the difference between adding/removing specific tokens, so it's unnecessary also to assume that the list was empty (which is incidentally the case)